backends/arm/arm_backend.py

```diff
@@ -145,8 +145,9 @@ def dbg_tosa_dump(tosa_fb, path):
         f.write(js)
 
 
-# Output to Vela with current file-based compilation
-# WARNING: if this changes, the runtime reader also needs to change
+# Output via Vela to binary stream for ArmBackendEthosU
+# WARNING: Do not change this without changing VelaBinStream.cpp as that
```

Comment:
once we figure out e2e testing, this should be tested

Reply:
I've structured the VelaBin* as having no dependencies and i have a local c test harness to enable a quick test of the serialisation form, would aim to structure it into unit test of serialisation.

Reply:
I meant AoT generates something which Runtime consumes and we verify the results - i.e. e2e test. So we don't have to worry about making sure AoT and runtime are aligned.